go.mod

```diff
@@ -48,3 +49,8 @@ require (
 	sigs.k8s.io/structured-merge-diff v0.0.0-20190130003954-e5e029740eb8 // indirect
 	sigs.k8s.io/yaml v1.1.0 // indirect
 )
+
+replace (
```

Comment:
We can get rid of this now that both repositories are public.

Reply:
Absolutely!